gustaf/show.py

```diff
@@ -2,6 +2,8 @@
 
 Everything related to show/visualization.
```

Comment:
Nice changes. I have to play around with the show options to get a hang of them. But they seem to be nice.

Reply:
I will add some notes about "what to expect" in PR